dhis-2/dhis-web-api-test/pom.xml

```diff
@@ -323,7 +323,7 @@
     <dependency>
       <groupId>org.springframework.security</groupId>
       <artifactId>spring-security-oauth2-core</artifactId>
-      <version>5.6.2</version>
+      <version>5.6.3</version>
```

Comment:
It looks like this is released together with the other spring security libraries so we should add it to the dependency management section using https://github.com/dhis2/dhis2-core/blob/271c51531b4c1a4cf883c71ab78e8e2afadce9e4/dhis-2/pom.xml#L74

What do you think @enricocolasante ?

Reply:
There are a few versions declared in the dhis-web-api-test pom file.
Maybe we can just merge this one and I can create a ticket to clean up the versions from here.
I see no reason why we would use different versions here as the rest of the application